tests/framework/installer/ceph_installer.go

```diff
@@ -45,7 +45,7 @@ const (
 	// test with the latest octopus build
 	octopusTestImage = "quay.io/ceph/ceph:v15"
 	// test with the latest pacific build
-	pacificTestImage = "quay.io/ceph/ceph:v16.2.6"
+	pacificTestImage = "quay.io/ceph/ceph:v16.2.7"
```

Comment:
Shall we just change this to the latest like with the octopus test tag?
```suggestion
	pacificTestImage = "quay.io/ceph/ceph:v16"
```

Reply:
Probably yes.

Reply:
And let's use 16.2 instead so we don't test on pre-release.

Reply:
v16 is expected to point to the latest v16 patch release, so currently is the same as v16.2.7. When would it be a prerelease?

Reply:
By pre-release I meant tags like 16.0 and 16.1, which are like alpha and beta tag IIRC.

Reply:
Sure, and since the v16 tag always points to the latest v16.2.x, the v16 and v16.2 tags are equivalent now.